pybossa/view/bloomberg.py

```diff
@@ -74,19 +74,25 @@ def handle_bloomberg_response():
     elif auth.is_authenticated:
         attributes = auth.get_attributes()
```

Comment:
Could we add a comment here stating what entering this block means? For example:
```python
# User is authenticated on BSSO, load user from GIGwork API.
```